tests/lib/acpi/integration/src/main.c

```diff
@@ -57,10 +59,12 @@ ZTEST(acpi, test_resource_enum)
 
 	zassert_not_null(dev, "Failed to get acpi device with given HID");
 
+	mmio_res.reg_base = reg_base;
 	ret = acpi_device_mmio_get(dev, &mmio_res);
 
 	zassert_ok(ret, "Failed to get MMIO resources");
 
+	irq_res.irqs = irqs;
 	ret = acpi_device_irq_get(dev, &irq_res);
```

Comment:
Aren't you missing the setting of the `*_max` values here? The `irq_res` and `mmio_res` variables are not currently initialised in this function, which would leave the ACPI function accessing uninitialised memory.